init/action.yml

```
@@ -1,5 +1,5 @@
 name: 'CodeQL: Init'
-description: 'Setup the CodeQL tracer'
+description: 'Setup CodeQL'
```

```suggestion
description: 'Set up CodeQL'
```

"setup" is a noun, and "set up" is a verb, although I think I'm in the minority in recognising this distinction...

---

I agree

---

@sampart - your comment makes me very happy 😄